web/turkey/survey/static/survey/js/auditors/mouse_movement_total.js

```diff
@@ -0,0 +1,26 @@
+var AuditorMouseMovementTotal = {
+    mouse_movement_total: 0,
+    previous_position: 0,
+    log_mousemove_event: function (e) {
+            this.mouse_movement_total += 1;
+        }
+    },
+    submit_callable: function () {
+        return {
+            'amount': this.mouse_movement_total
+        };
+    }
+};
+
+var auditor_mouse_movement_total = Object.create(AuditorMouseMovementTotal);
+
+$(document).ready(function() {
+    $(window).mousemove(auditor_mouse_movement_total
```

Comment:
This code will crawl.[ MDN ](https://api.jquery.com/mousemove/)indicates that 

> Keep in mind that the mousemove event is triggered whenever the mouse pointer moves, even for a pixel. This means that hundreds of events can be generated over a very small amount of time. If the handler has to do any significant processing, or if multiple handlers for the event exist, this can be a serious performance drain on the browser. It is important, therefore, to optimize mousemove handlers as much as possible, and to unbind them as soon as they are no longer needed.

Suggest only recording event if (x) amount of time has passed, or polling mouse position at regular intervals.